back/app/policies/idea_policy.rb

```diff
@@ -58,9 +58,13 @@ def by_slug?
     show?
   end
 
+  def draft_by_phase?
+    show?
```

Comment:
```suggestion
    show? && owner?
```
I believe it won't affect how the action currently works, but still may be clearer and maybe more reliable for the future.